throughput/spam.go

```diff
@@ -208,6 +213,13 @@ func (s Spammer) broadcast(
 						utils.Outf("{{cyan}}skipping issuance because large backlog detected, decreasing target tps:{{/}} %d\n", currentTarget)
 					} else {
 						utils.Outf("{{cyan}}skipping issuance because large backlog detected, cannot decrease target{{/}}\n")
+						consecutiveFailedDecreaseTarget++
+						if consecutiveFailedDecreaseTarget >= failedRunsToDecreaseTarget {
+							utils.Outf("{{cyan}}skipping issuance because large backlog detected, exiting{{/}}\n")
+							broadcastErr = ErrLargeBacklog
+							stop = true
+							break
+						}
```

Comment:
The changes here seem to be a change unrelated goal of removing the funds map... should these be a separate PR?